phantom-scrape.js

```diff
@@ -72,7 +72,13 @@ if (userAgent) {
 page.settings.loadImages = false;
 
 // ensure we don't waste time trying to load slow/missing resources
-page.settings.resourceTimeout = 1000;
+page.settings.resourceTimeout = 5000;
+
+// if we do timeout a slow resource, say something useful
+page.onResourceTimeout = function(request) {
+    console.log('Response (#' + request.id + '): ' + JSON.stringify(request));
```

Comment:
Oh yes, I see what you mean. Getting rid of the exit altogether might be better.